### ISSUE TITLE
[Bug]: On Mac Silicon Designed for iPad (target) OneSignal popups are wrong size - bigger then window

### ISSUE THREAD

**Issue by sabiland** [OneSignal/OneSignal-iOS-SDK#1259]
### What happened?

For Designed for iPad target for Mac Silicon all OneSignal (rating, new feature, apn) popups are wrong size. 

Example:
<img width="1057" alt="Screenshot 2023-04-28 at 15 36 23" src="https://user-images.githubusercontent.com/12098915/235162437-70ee7c9d-adbf-4f87-a8b6-274c6328d6e8.png">

### Steps to reproduce?

```Markdown
1. Make Mac Silicon Designed for iPad version and run it. You'll see popup sizes are wrong. Not centered inside window.
```

### What did you expect to happen?

I expected Popup would be nicely centered as is on iPhone/iPad.

### OneSignal iOS SDK version

main

### iOS version

15

### Specific iOS version

```Markdown
iOS 16
Macbook Pro M1
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by emawby** [comment]
Thank you for reporting we will investigate! On other build targets (mac catalyst) do the popups show properly?

**Comment by sabiland** [comment]
> Thank you for reporting we will investigate! On other build targets (mac catalyst) do the popups show properly?

Yes, they show proper on iPhone and iPad. Though it could be smaller on iPad, not full screen ;)

**Comment by nan-li** [comment]
Hi, we have had no further reports of this. Please upgrade the OneSignal SDK if you or anyone is still having this issue. If this is still an issue, please open a new report with updated information.
